chef/cookbooks/neutron/recipes/common_agent.rb

```diff
@@ -202,10 +203,37 @@
       command "ovs-vsctl add-br #{name}; ip link set #{name} up"
       not_if "ovs-vsctl list-br |grep -q #{name}"
     end
-    execute "Neutron: add #{bound_if} to #{name}" do
-      command "ovs-vsctl del-port #{name} #{bound_if} ; ovs-vsctl add-port #{name} #{bound_if}"
-      not_if "ovs-dpctl show system@#{name} | grep -q #{bound_if}"
```

Comment:
This ```not_if``` command (and all the copied versions of that below) is wrong (there is no ```system@br-fixed``` in recent openvswitch releases). Better use: ```ovs-vsctl ifcase-to-br #{bound_if} | grep -q #{name}```